### PR TITLE
Pma 695 integration script

### PR DESCRIPTION
**Pull request by MatteoWebDeveloper**

### CODE REVIEW

**Review by andrew-james-yoti — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 node_modules
+dist
 .DS_Store
 *.pem
-package-lock.json
\ No newline at end of file
+package-lock.json
```
Comment (christoffee3) [inline .gitignore:5]: was you meant to ignore package-lock ?

File: browser/src/main.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
+class PMA {
+  /**
+   * @param {object} configuration - An object
+   * @param {string} configuration.brandId - An string with the client brand id
+   * @param {string} configuration.redirectTo - url client app, it will be used to redirect at the end of the flow
+   * @param {string} configuration._domain - change iframe and location domain only for testing purposes
+   */
+  constructor({ brandId, redirectTo = window.location, _domain = process.env.DOMAIN }) {
```
Comment (christoffee3) [inline browser/src/main.js:8]: do we want the `redirectTo` to default rather than fail ?

Comment (christoffee3) [reply browser/src/main.js:8]: also `_domain` or are they optional ?

File: .gitignore (hunk 1 above)
Comment (MatteoWebDeveloper) [reply .gitignore:5]: yes package lock is not needed, the script is going to be build and upload on a cdn

File: browser/src/main.js (hunk 2 above)
Comment (MatteoWebDeveloper) [reply browser/src/main.js:8]: `redirectTo` is mandatory.
`_domain` is optional but also is a private property and you can change it at your own risk

**Review by andradaolah — APPROVED** [review]

**Pull request merged**